open-reac/README.md

```diff
@@ -1,62 +1,402 @@
+data and the parameters of the network on which the reactive OPF is executed.
+These files are obtained by using the
+[AMPL export of PowSyBl](https://github.com/powsybl/powsybl-core/blob/main/ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkWriter.java).
+
+#### 3.2 Configuration of the run
+
+The user can configure the run with the dedicated Java interface 
+(see [OpenReacParameters](src/main/java/com/powsybl/openreac/parameters/input/OpenReacParameters.java)).
+Specifically, the user can set various parameters and thresholds used in the preprocessing and modeling of the reactive OPF. 
+These are specified in the file `param_algo.txt`:
+
+| Parameter                          | Description                                                                                                                                                                       | Default value     | Domain                                        |
+|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|-----------------------------------------------|
+| `log_level_ampl`                   | Level of display for AMPL prints                                                                                                                                                  | INFO              | {DEBUG, INFO, WARNING, ERROR}                 |
+| `log_level_knitro`                 | Level of display for solver prints (see [AMPL documentation](https://dev.ampl.com/ampl/options.html)                                                                              | $1$               | {0, 1, 2}                                     |  
+| `objective_choice`                 | Choice of the objective function for the ACOPF (see [7](#7-alternative-current-optimal-power-flow))                                                                               | $0$               | {0, 1, 2}                                     |
+| `ratio_voltage_target`             | Ratio to calculate target V of buses when `objective_choice` is set to $1$ (see [7](#7-alternative-current-optimal-power-flow))                                                   | $0.5$             | $\[0; 1\]$                                    |
+| `coeff_alpha`                      | Weight to favor more/less minimization of active power produced by generators or deviation between them and target values (see [6.2](#62-alternative-current-optimal-power-flow)) | $1$               | $\[0; 1\]$                                    |
+| `Pnull`                            | Threshold of active and reactive powers considered as null                                                                                                                        | $0.01$ (MW)       | $\[0; 1\]$                                    |
```

Comment:
I cannot find Pnull nor Znull parametrization in OpenReacParameters but I guess they will be added in #43

Reply:
Yes, exactly :)